scripts/generate-policy-config.py

```diff
@@ -64,11 +65,10 @@
         if entry.name == config_filename:
             with open(entry.path) as config_file:
                 config = yaml.safe_load(config_file)
-                if config['deploymentMode'] == 'Policy':
```

Comment:
and it just works? I guess the rest of the script was already prepared for that...

Reply:
there aren't many occurrences, so it just worked. pure luck that we could avoid a "migration".